src/proto_conversions.cc

```diff
@@ -265,6 +265,7 @@ bool Options_Format_Enum_Parse(const std::string& format, Options::Format* f) {
       {"json", Options::json},
       {"gpx", Options::gpx},
       {"osrm", Options::osrm},
+      {"pbf", Options::pbf},
```

Comment:
missed this bug in a previous pr. the result of not having this is that if you send a json request but you want a pbf response it would still send you json. pbf request with pbf response worked fine though.

Reply:
is that worth a small test? (didn't check, maybe there is one now?)